### ISSUE TITLE
Screen Reader NVDA 2015.2 reads the heading twice

### ISSUE THREAD

**Issue by TheLast** [angular-ui/bootstrap#4276]
The current day/month/year heading with ARIA tags indicating a live region (role="heading", aria-live="assertive", aria-atomic="true") is being read twice sometimes:
![nvda_datepicker](https://cloud.githubusercontent.com/assets/4096190/9493210/5ddf834a-4bfe-11e5-9586-627f59731eda.png)

**Comment by wesleycho** [comment]
Hi @TheLast, do you know what is the culprit here?

**Comment by ssmirnow** [comment]
hi @wesleycho, 
I have the same problem. At first I thought that it's somehow related to Angular (i.e. twice setting of scope.title var), but adding a watcher to test if that's the problem showed that setter is called only once,  so I believe that's a problem somehow related to aria-roles.

**Comment by TheLast** [comment]
Hi @wesleycho and @ssmirnow 
Here's an example of accessible datepicker with the same ARIA roles: oaa-accessibility.org/example/15/ 
Still difficult to say if it's not a fault of ARIA roles working with angular

**Comment by icfantv** [comment]
Closing in favor of #4772.
